test/jdk/com/sun/net/httpserver/bugs/8300268/MaxIdleConnectionsTest.java

```diff
@@ -105,11 +105,10 @@ public void test() throws Exception {
             System.out.println("Received successful response for request " + i);
         }
 
-        // assert that the expected number of idle connections has been reached
-        // meaning the extra connection was served but then closed
+        // assert that the limit set by maxIdleConnections was not exceeded
         int idleConnectionCount = HttpServerAccess.getIdleConnectionCount(server);
         System.out.println("count " + idleConnectionCount);
-        assertEquals(maxIdleConnections, idleConnectionCount);
+        assertTrue(maxIdleConnections >= idleConnectionCount);
```

Comment:
It would be good to add an error message showing the value of both quantities. Pity that there is no `assertLte` or `assertGte`.